docs/source/tests.matrices.rst

```diff
@@ -1,29 +0,0 @@
-tests.matrices package
```

Comment:
similar argument as for `setup.rst`, we don't care about documenting our test code.